app/src/main/java/com/xmartlabs/daydreaming/ui/LoginRegisterFragment.java

```diff
@@ -25,7 +26,13 @@
  * Created by chaca on 4/12/17.
  */
 @FragmentWithArgs
-public class LoginRegisterFragment extends BaseFragment{
+public class LoginRegisterFragment extends BaseFragment {
+  private static final int MAX_PASSWORD_LENGTH = 5;
+  @StringRes
+  private static final int PASSWORD = R.string.user;
```

Comment:
Yes, because the validation through a authentication server was left out of the main scope.